shared/profile/friendships.desktop.js

```diff
@@ -108,13 +108,11 @@ class FriendshipsRender extends Component<Props> {
             this.props.onSwitchTab && this.props.onSwitchTab('Followers')
           }}
         >
-          <Box style={{marginTop: globalMargins.small}}>
+          <Box style={{marginTop: globalMargins.large}}>
```

Comment:
This also applies to the friendship list when it's not empty, is that intended?

<img width="797" alt="image" src="https://user-images.githubusercontent.com/11968340/36558564-62503efc-17d9-11e8-8a64-2a6eecf63631.png">

Reply:
Woops, nope.